### PR TITLE
removes a MLP reference [is #12431]

### PR DESCRIPTION
**Pull request by ghost**
Closes #12431.
## Changelog
:cl: zamolxius & Elephant Seal
fix: fixed a My Little Pony: Friendship is Magic reference in the codebase
/:cl:

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
You can just set it up so that it closes:#prnum so that it'd do it autonomously via git keyword but its good that you took the user feedback of that thread on board.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1952,13 +1952,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
 	color = "#ff6633"
 	boozepwr = 20
-	taste_description = "an honest day's work at the orchard"
+	taste_description = "like you should shoot some redskins"
 	glass_icon_state = "applejack_glass"
 	glass_name = "Applejack"
-	glass_desc = "You feel like you could drink this all neight."
+	glass_desc = "The name applejack derives from the traditional method of producing the drink, jacking, the process of freezing fermented cider and then removing the ice, increasing the alcohol content. Alcoholic fruit beer produced after the fall harvest was left outside during the winter. Periodically the frozen chunks of ice which had formed were removed, thus concentrating the unfrozen alcohol in the remaining liquid. Starting with the fermented juice, with an alcohol content of less than ten percent, the concentrated result can contain 25–40% alcohol. Because freeze distillation is a low-infrastructure method of production compared to evaporative distillation, and does not require the burning firewood to create heat, hard cider and applejack were historically easy to produce, though more expensive than grain alcohol."
```
Comment (TrickyDickRichard) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: i would condense this, probably to just the first sentence

**Comment by Toxici11i** [comment]
/tg/ moment

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (ghost) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: No

**Review by ghost — DISMISSED** [review]
This is a based PR.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (TrickyDickRichard) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: why, the way it is now is chatspammy

**Comment by ghost** [comment]
+1 this is in the spirit of Hippie

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (TrickyDickRichard) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: make it less chat spammy god dammit it's literally just a whole wikipedia page paragraph

Comment (ghost) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: the beta tg sentence vs the CHAD HIPPIE INFODUMP

Comment (Toxici11i) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: stop being a faggot, the shotgun has an entire copypasta as its description.

Comment (TrickyDickRichard) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: > stop being a faggot, the shotgun has an entire copypasta as its description.

because it's a copypasta faggot
just change it to something like this 
The name applejack is derived from the traditional method of producing the drink, jacking, which refers to the practice of freeze-distilling, which was used to make the beverage. This method of distilling meant that the drink historically easier, but the product was more expensive than grain alcohols.

Comment (Toxici11i) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: aka "its in the spirit of hippie" or "stay mad you inane faggot"

**Comment by TrickyDickRichard** [comment]
-1 until the desc gets changed call me mad again

**Comment by Toxici11i** [comment]
+1 fuck trickydick hes an unfun faggot

**Comment by mikolaj007** [comment]
+0, pointless PR, but im not gonna stop you

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (ghost) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: TrickyDick is wrong. @zamolxius, leave the whole article. Dick is outvoted.

**Comment by ghost** [comment]
Think that other PR had enough approvals, merg time

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -1952,13 +1952,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 /datum/reagent/consumable/ethanol/applejack
 	name = "Applejack"
-	description = "The perfect beverage for when you feel the need to horse around."
+	description = "Applejack is a strong apple-flavored alcoholic drink produced from apples. Popular in the American colonial era, the drink's prevalence declined in the 19th and 20th centuries amid competition from other spirits."
 	color = "#ff6633"
 	boozepwr = 20
-	taste_description = "an honest day's work at the orchard"
+	taste_description = "like you should shoot some redskins"
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: "You can taste" is the message that pops up when you taste something. The taste_description here should be changed to make more sense.

**Review by ghost — DISMISSED** [review]

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 2 above)
Comment (TrickyDickRichard) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: i think it was supposed to say "you feel like you could shoot some redskins" but i guess that was for the description actually just make it say "apple cider with added death"

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (TrickyDickRichard) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: im not outvoted for shit neither of these other people are designers

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 3]
```diff
@@ -1148,7 +1148,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 /datum/reagent/consumable/ethanol/hcider
 	name = "Hard Cider"
-	description = "Apple juice, for adults."
+	description = "juice, for adults."
```
Comment (TrickyDickRichard) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: why

Comment (ghost) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: ah,  accident

**Comment by TrickyDickRichard** [comment]
i dont know how to use github is the description changed

**Comment by ghost** [comment]
ill do it tomorrow

**Comment by ghost** [comment]
merge

**Review by Manly-Dwarf — CHANGES_REQUESTED** [review]
why is there an entire wikipedia paragraph
tl;dr you plagiarist

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 4]
```diff
@@ -1952,13 +1952,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
 	color = "#ff6633"
 	boozepwr = 20
 	taste_description = "an honest day's work at the orchard"
 	glass_icon_state = "applejack_glass"
 	glass_name = "Applejack"
-	glass_desc = "You feel like you could drink this all neight."
+	glass_desc = "The name applejack derives from the traditional method of producing the drink, jacking, the process of freezing fermented cider and then removing the ice, increasing the alcohol content. Alcoholic fruit beer produced after the fall harvest was left outside during the winter. Periodically the frozen chunks of ice which had formed were removed, thus concentrating the unfrozen alcohol in the remaining liquid. Starting with the fermented juice, with an alcohol content of less than ten percent, the concentrated result can contain 25–40% alcohol. Because freeze distillation is a low-infrastructure method of production compared to evaporative distillation, and does not require the burning firewood to create heat, hard cider and applejack were historically easy to produce, though more expensive than grain alcohol."
```
Comment (Manly-Dwarf) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	glass_desc = "Applejack is created by distilling cider via a method known as jacking, where the beverage is frozen and the ice is removed, leaving only the alcoholic content."
```
"hey can i copy your homework"
"yeah just change it so the teacher can't tell"

**Comment by TrickyDickRichard** [comment]
> why is there an entire wikipedia paragraph
> tl;dr you plagiarist

exactly look at these nogs

**Review by Manly-Dwarf — APPROVED** [review]

**Comment by TrickyDickRichard** [comment]
+1 epic now

**Comment by ghost** [comment]
@Manly-Dwarf that's the joke you cancerous bollock

**Comment by boodaliboo** [comment]
+1 hahahahha p[acman

**Comment by ghost** [comment]
alright, merging

**Comment by Toxici11i** [comment]
hasbro gonna finna sue us for copyright infingement on the applejack icon btw
